hoomd/mpcd/Integrator.cc

```diff
@@ -81,6 +81,7 @@ void mpcd::Integrator::update(unsigned int timestep)
         m_mpcd_comm->communicate(timestep);
     #endif // ENABLE_MPI
 
+    bool checkcollide = checkCollide(timestep);
```

Comment:
Use underscore to make more readable.
```suggestion
    bool check_collide = checkCollide(timestep);
```
This also needs to move above, before line 73, and replace **all** calls to `checkCollide` with `check_collide`.